### ISSUE TITLE
Docs for generate-shell-autocompletion do not work with some existing profiles

### ISSUE THREAD

**Issue by mtnpke** [astral-sh/uv#8923]
The docs on generate-shell-autocompletion at https://docs.astral.sh/uv/getting-started/installation/#shell-autocompletion list this command for powershell:

```
Add-Content -Path $PROFILE -Value '(& uv generate-shell-completion powershell) | Out-String | Invoke-Expression'
```

This fails if

* $PROFILE already exists and
* does not end with a newline.

In this case, the line will be added to the end of the last line, resulting in a broken profile file.

A more reliable replacement could be:

```
Add-Content -Path $PROFILE -Value "`r`n", '(& uv generate-shell-completion powershell) | Out-String | Invoke-Expression'
```

**Comment by charliermarsh** [comment]
One downside of the above, I think, is that we'd add a _leading_ newline if the file didn't exist already?

**Comment by mtnpke** [comment]
That is correct. Unfortunately, I haven't found a better solution that does not add the leading newline and is a reasonably compact one-liner.

**Comment by JonasR** [comment]
Ran into this today and had no idea what was going on. Took me a bit to google this issue as well, but thanks for pointing me at the underlying cause and providing a fix.

The error I got was
`uv completion Cannot convert 'System.Object[]' to the type 'System.String' required by parameter 'Command'. Specified method is not supported.`

**Comment by SnowFox4004** [comment]
Found the same problem today. And adding a newline does fix the problem. Thanks for giving solutions
